src/DocChecks.jl

```diff
@@ -140,11 +153,12 @@ function doctest(block::Markdown.Code, meta::Dict, doc::Documents.Document, page
             kwargs = Meta.parse("($(lang[nextind(lang, idx):end]),)")
             for kwarg in kwargs.args
                 if !(isa(kwarg, Expr) && kwarg.head === :(=) && isa(kwarg.args[1], Symbol))
+                    file = meta[:CurrentFile]
+                    lines = find_codeblock_in_file(block.code, file)
```

Comment:
It should have trouble finding the line numbers in CRLF files though, right? Because `block.code` has the newlines normalized? I am wondering if we should just move the normalization into the `eval_*` functions and not modify the `.code` field at all.